cmd/cmd.go

```diff
@@ -18,6 +18,10 @@ package main
 
 import (
 	"context"
+	"fmt"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"net"
+	"net/http"
 	"os"
```

Comment:
pls sort the import block. [Contributing](https://github.com/CECTC/dbpack/blob/dev/CONTRIBUTING.md) there is a guide.